src/cryptoadvance/specterext/notifications/__main__.py

```diff
@@ -0,0 +1,51 @@
+from cryptoadvance.specter.cli import entry_point
```

Comment:
I think we should delete this file

Reply:
Done https://github.com/cryptoadvance/specter-desktop/pull/1766/commits/2de3e1e20677fe1578d315e77e952c41304adf5d